contracts/contracts/FleekERC721.sol

```diff
@@ -86,6 +84,7 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl, Fl
     function initialize(string memory _name, string memory _symbol) public initializer {
         __ERC721_init(_name, _symbol);
         __FleekAccessControl_init();
+        _appIds = 0;
```

Comment:
Do we need to initialize also in the AccessControl side?

Reply:
In mappings, the default value is always zero.